sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedParquetRecordReader.java

```diff
@@ -50,6 +50,9 @@
  * TODO: make this always return ColumnarBatches.
  */
 public class VectorizedParquetRecordReader extends SpecificParquetRecordReaderBase<Object> {
+  // TODO: make this configurable.
+  private static final int CAPACITY = 4 * 1024;
```

Comment:
Do we want to have a consistent default capacity for `OrcColumnarBatchReader` and `VectorizedParquetRecordReader`, or have them set separately?

Reply:
we should set them separately, places using `ColumnarBatch` should decide the default size themselves.

Reply:
Then, what should we use for capacity in `ColumnVectorUtils.toBatch()` ?

Reply:
I also pick `4 * 1024`, but that method would be removed eventually, it's not used anywhere, just an example for writing rows to batches.

Reply:
If this is just an example, how about move it to the ColumnarBatchSuite?

Reply:
but it's not a test, it's an example about how to implement a `ColumnarBatch` producer. We would remove it when we publish `WritableColumnVector` and have a stable API for writing `ColumnarBatch`.